package.json

```diff
@@ -34,6 +34,7 @@
     "@ckeditor/ckeditor5-theme-lark": "~35.1.0",
     "@ckeditor/ckeditor5-upload": "^35.1.0",
     "@ckeditor/ckeditor5-vue2": "^3.0.1",
+	"@ckeditor/ckeditor5-html-embed": "~35.1.0",
```

Comment:
the corresponding package-lock.json change is missing here

Reply:
@ChristophWurst 
I've added the missing entries in the package-lock.json. Sadly i used the wrong account to push this into my repo. I do hope this is not a problem.

Reply:
@ChristophWurst 
Hey there, is everything now to your liking and correct?

Reply:
I believe the only thing left to fix is indentation. Look at the file you changed and existing lines before/after. Use the same indentation style.

Reply:
@ChristophWurst 
I fixed what looked weird to me.
I dont see anything else, if you see something please let me know.